packages/paypal-js/types/v6/components/find-eligible-methods.d.ts

```diff
@@ -0,0 +1,75 @@
+import {
+    PayLaterCountryCodes,
+    PayLaterProductCodes,
+    PayPalCreditCountryCodes,
+} from "./paypal-payments";
+
+export type EligiblePaymentMethods =
+    | "paypal_pay_later"
+    | "paypal_credit"
+    | "paypal"
+    | "venmo";
+
+export type FindEligibleMethodsOptions = {
+    currencyCode?: string;
+};
+
+export type FundingSource =
```

Comment:
can we reduce this list to just what we support today with this first round of types? 

- paypal
- paylater
- credit